manifests/install.pp

```diff
@@ -8,6 +8,12 @@
     ensure => $candlepin::version,
   }
 
+  if $facts['selinux'] {
```

Comment:
does this eval to true if selinux is available, or enabled?

Reply:
Available afaict.

```sh
[vagrant@centos7-devel foreman]$ sudo setenforce 0
[vagrant@centos7-devel foreman]$ facter selinux
true
```

Reply:
There's also `$facts['selinux_enforced']`. Should we check that?

Reply:
yeah, i think we should install regardless of enforcing or permissive mode

Reply:
> There's also `$facts['selinux_enforced']`. Should we check that? It seems to me that we should install even if they have selinux disabled. That way if they enable selinux, the proper policy is already there.

I agree, it should be installed if selinux is disabled or permissive.

Reply:
IMHO we should install if it's selinux "enabled". Whether it's enforcing or permissive is not that relevant. If it's in permissive mode then you don't want to get spammed by denials either. That means this is the correct fact to use.